ani-cli

```diff
@@ -237,13 +250,14 @@ download() {
 }
 
 play_episode() {
+    [ "$skip_intro" = 1 ] && get_skip_times
```

Comment:
Change it to  `[ "$skip_intro" = 1 ] && skip_flag="$(ani-skip "$title" "$ep_no")"`

Reply:
changed to:
```sh
[ "$skip_intro" = 1 ] && skip_flag="$(ani-skip "$title" "$ep_no" | cut -d' ' -f1)" && skip_arg="$(ani-skip "$title" "$ep_no" | cut -d' ' -f2)"
```